print_format/request_for_quotation.jinja

```diff
@@ -33,7 +39,7 @@
             {%- endif %}
             </p>
         </div>
-        {{ doc.vendor }}<br />
+        {{ doc.vendor or supplier_doc.name }}<br />
```

Comment:
```suggestion
        {{ supplier_doc.name }}<br />
```

When `doc.vendor` is available, `supplier_doc.name` will also be available and identical.